velox/functions/sparksql/Register.cpp

```diff
@@ -53,6 +54,11 @@ extern void registerElementAtFunction(
     const std::string& name,
     bool enableCaching);
 
+inline void registerFromBase64Function(const std::string& prefix) {
+  registerFunction<FromBase64Function, Varbinary, Varchar >(
```

Comment:
Looks no need to define `registerFromBase64Function`.

I note in Spark unbase64 & hex/unhex are in the same category. So better to register it here:
https://github.com/facebookincubator/velox/blob/93b599b94c644874957a7371c0f365345851d831/velox/functions/sparksql/RegisterArithmetic.cpp#L58